x-pack/plugins/security/server/routes/analytics/authentication_type.test.ts

```diff
@@ -312,21 +325,95 @@ describe('POST /internal/security/analytics/_record_auth_type', () => {
       });
 
       const response = await routeHandler(getMockContext(), request, kibanaResponseFactory);
+
       expect(response.status).toBe(200);
+
       expect(response.payload).toEqual({
         timestamp: initialTimestamp,
         signature: '46d5841ad21d29ca6c7c1c639adc6294c176c394adb0b40dfc05797cfe29218e',
       });
+
       expect(response.payload.signature).not.toEqual(initialSignature);
 
       expect(routeParamsMock.analyticsService.reportAuthenticationTypeEvent).toHaveBeenCalledTimes(
         1
       );
+
       expect(routeParamsMock.analyticsService.reportAuthenticationTypeEvent).toHaveBeenCalledWith({
         authenticationProviderType: 'http',
         authenticationRealmType: 'native',
         httpAuthenticationScheme: 'Custom',
       });
     });
   });
+
+  describe('logApiKeyWithInteractiveUserDeprecated', () => {
+    it('should log a deprecation warning if interactive user is using API Key', async () => {
```

Comment:
"interactive user" is not a term I normally see.  Can we drop "interactive" or find another way to explain what is meant here?

Reply:
How about something like:

> Users authenticating via API key while accessing Kibana through a web browser is deprecated and will not be supported in a future version

Reply:
How about something like:

> should log a deprecation warning if a user is authenticating via API key while accessing the application through a web browser